src/validations/request.validation.js

```diff
@@ -12,38 +10,38 @@ import RequestService from '../services/request.service';
  * @returns {validation} this function validate request route
 */
 export async function requestValidation(req, res, next) {
-  const signInUser = JwtService.verifyToken(req.headers.authorization);
-  const { userId } = req.params;
+  const page = parseInt(req.query.page, 10);
+  const limit = parseInt(req.query.limit, 10);
 
   const schema = Joi.object({
-    userId: Joi.string().max(11).regex(/^[0-9]{1,11}$/)
+    page: Joi.number().greater(0).required()
       .messages({
-        'string.max': 'User ID length must be less than or equal to 11 characters long',
-        'string.pattern.base': 'Invalid, ID must contain only numbers'
+        'number.greater': 'Page must be greater than 0',
+        'object.unknown': `${page} is not allowed`,
+        'any.required': 'Page is required',
+        'number.base': 'Page must be a number'
+      }),
+    limit: Joi.number().greater(0).required()
+      .messages({
+        'number.greater': 'Limit must be greater than 0',
+        'object.unknown': `${limit} is not allowed`,
+        'any.required': 'Limit is required',
+        'number.base': 'Limit must be a number'
       })
-  });
+  }).options({ abortEarly: false });
 
-  const { error } = schema.validate(req.params);
+  const { error } = schema.validate(req.query);
   if (error) {
-    ResponseService.setError(400, error.details[0].message);
+    const errMessages = [];
+    error.details.map((err) => errMessages.push(err.message));
```

Comment:
Why are you pushing to a new array? Already gave feedback on the use of .map function.